app/models/DepartureMessageNotification.scala

```diff
@@ -47,12 +53,16 @@ object DepartureMessageNotification {
         obj ++ Json.obj("requestId" -> requestId(departure.departureId))
     }
 
-  def fromRequest(request: DepartureResponseRequest[NodeSeq], timestamp: LocalDateTime): DepartureMessageNotification =
+  def fromRequest(request: DepartureResponseRequest[NodeSeq], timestamp: LocalDateTime): DepartureMessageNotification = {
+    val messageId    = MessageId.fromIndex(request.departure.messages.length)
+    val departureUrl = requestId(request.departure.departureId)
     DepartureMessageNotification(
+      s"$departureUrl/messages/${messageId.index}",
```

Comment:
Same question as on the other PR - should this be the 1-indexed ID that is created by the `unapply`?

Perhaps we could refactor `MessageId` so that it has a method for getting the user-facing ID rather than an `unapply` method which is a bit magical IMO?